src/app/forms/frontcountry-cabins/frontcountry-cabins.component.scss

```diff
@@ -0,0 +1,9 @@
+@import "src/assets/themes/_variables.scss";
+
+.grey-bg {
```

Comment:
we could write this as:

```
.form-section {
    padding: 2 rem;
}

.form-section:nth-child(odd) {
    background-color: $form-grey;
}
```

And move it to `base.scss`. Then the css will auto-alternate the background colours for each `form-section` - which we can then easily copy across all forms.

Reply:
Addressed in https://github.com/bcgov/bcparks-ar-admin/commit/81f30ebcd0b23f51d1621bc5f208ec011e1bc7af